wherehows-web/config/environment.js

```diff
@@ -74,6 +74,10 @@ module.exports = function(environment) {
 
     ENV.APP.rootElement = '#ember-testing';
     ENV.APP.autoboot = false;
+    ENV['ember-cli-mirage'] = {
```

Comment:
Will this be enabled and autostart in `PROD` and `DEV`?

Reply:
Just test.